modules/tinymce/src/plugins/media/test/ts/browser/ContentFormatsTest.ts

```diff
@@ -37,7 +37,7 @@ describe('browser.tinymce.plugins.media.ContentFormatsTest', () => {
   it('TBA: Embed retained as is', () => {
     const editor = hook.editor();
     editor.setContent(
-      '<embed src="320x240.ogg" width="100" height="200">text<a href="#">link</a></embed>'
+      '<p><embed src="320x240.ogg" width="100" height="200">text<a href="#">link</a></embed></p>'
```

Comment:
Why was this needed? This definitely doesn't seem right as the root is not a transparent element, it's an inline element.

Reply:
Rolled this back. Tried to avoid having one input and another output of things. Not really related to this PR but in general.